README.md

```diff
@@ -9,7 +9,7 @@
 [![Slack channel](https://img.shields.io/badge/chat-slack-green.svg?logo=slack)](https://kotlinlang.slack.com/messages/coroutines/)
```

Comment:
The Kotlin version shield also needs to be updated.

Reply:
Done